VMAccess/vmaccess.sh

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env bash
+
+COMMAND="./vmaccess.py"
+PYTHON=""
+
+USAGE="$(basename "$0") [-h] [-i|--install] [-u|--uninstall] [-d|--disable] [-e|--enable] [-p|--update] -- program to invoke the extension script
+
+where:
+    -h              show this help text
+    -i|--install    install the extension
+    -u|--uninstall  uninstall the extension
+    -d|--disable    disable the extension
+    -e|--enable     enable the extension
+    -p|--update     update the extension
+"
+
+function find_python(){
+    local python_version=$1
+
+    # Check if there is python defined.
+    which python > /dev/null 2>&1
```

Comment:
There are some posts (e.g. https://stackoverflow.com/questions/592620/how-to-check-if-a-program-exists-from-a-bash-script) discouraging the use of which to check for the existence of a command; apparently some implementations don't set an exit code. Consider using command instead:
```

if command -v python; then
...
fi
```